eng/Version.Details.xml

```diff
@@ -1,12 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="8.0.0-alpha.1.23214.4">
+      <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
+      <Sha>05b4fdac346e01716abda4c92fcf0b7880e73792</Sha>
+      <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
+    </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
     <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="8.0.0-beta.23174.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>1177b37e60c7271ab461098aefbdd03edc15b90d</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
+    <Dependency Name="Microsoft.SourceLink.GitHub" Version="1.2.0-beta-23172-02" CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk">
```

Comment:
Looks like a potential version mismatch given the prebuilts reported in CI.

Reply:
Will check - I used the version that matches prebuilts reported in this repo's source-build.

Reply:
> Looks like a potential version mismatch given the prebuilts reported in CI.

Fixed with https://github.com/dotnet/cecil/pull/77/commits/901c0f9573d90e25ab2f6d45dc680b16337130fc

This was caused by Arcade update that happened 30 minutes ago, after my local build, but before PR was created - the timing...